templates/common/infra/bicep/core/database/cosmos/cosmos-account.bicep

```diff
@@ -9,6 +9,8 @@ param keyVaultName string
 @allowed([ 'GlobalDocumentDB', 'MongoDB', 'Parse' ])
 param kind string
 
+param disableLocalAuth bool = false
```

Comment:
Yes, it's now supported (except for Mongo vCore currently, which isn't covered by this template anyways)

Reply:
Okay, I think in order to support this we'd need to update the templates to use MI instead of connection strings.  Do you have a link to the doc that says Cosmos now supports MI for Mongo?  I can't seem to find it.

Reply:
Digging a bit in the docs, it seems cosmos/mongo supports RBAC but not disabling local auth, as seen in the limitations section here: https://learn.microsoft.com/en-us/azure/cosmos-db/mongodb/how-to-setup-rbac#limitations

I've previously seen this docs https://learn.microsoft.com/en-us/azure/cosmos-db/role-based-access-control and assumed it was available for MongoDB (not vCore) too. I'll update the PR to only surface the parameter for NoSQL

Reply:
Thanks, but I don't think we'll take this change in infra/core since we are moving to AVM.